### PR TITLE
New PR to fix missing items

### PR DESCRIPTION
**Pull request by p3dstore**
* [x] The mod, firmware configuration or slicer profile is in the correct category
    folder. Printable mods go to `printer_mods/`, firmware configurations
    go to `firmware_configurations/`, slicer profiles go to `slicer_profiles/`.
    Create a subfolder with your name, and place the mods in a subfolder with
    a descriptive name within that folder, e.g.: `/printer_mods/FHeilmann/flux_capacitor`
  * [x] Folders names MUST NOT contain spaces. If possible, make sure file names also 
    do not contain any spaces.
  * [x] For each mod, add a small `README.md` file to its folder with a short description
    of what the mod accomplishes. This readme can be used to add pictures, give assembly
    instructions or specify a bill of materials if the mod requires additional hardware.
  * [x] The PR modifies the top-level `README.md` of the category folder adding the 
    contribution to the table. Read the top part of the file for instructions on how
    to do this. Please preserve the alphabetical ordering while adding new rows. Make sure
    to fill out the compatibility matrix to indicate which versions of the Voron printer
    the submission is compatible with.
  * [x] The mod/configuration/profile has been tested by the person submitting the mod 
    and/or other Voron users. Make sure to add information about how the mod was tested below. 
  * [x] The mod is not merely a slight modification of an official Voron part, configuration 
    or profile (i.e. an official Voron part with a few mm added or removed or a slicer profile 
    which only modifies a few values). *(When in doubt, contact one of the admins in the 
    Voron discord before submitting the PR)*
  * [ ] Submitted STLs are printable without support. *(If the mod does not meet this criterion
    join the Voron discord and ask the other users for advice on how to modify the mod such 
    that it does not require supports)*
  * [ ] Submitted STL files are not corrupt. *(This can be tested by opening the STL in PrusaSlicer
    and checking if mesh errors are reported.)*
  * [ ] Submitted STL files are oriented and scaled properly for printing.
  * [x] Submitted firmware configs or slicer profiles contain no sensitive data (e.g. API keys).

#### Which mods/configurations/profiles are added by this PR?
Added my firmware configs and slicer profile.
#### How was it tested? 
Printing my keyboard cases for my business.
#### Any background context you want to provide?
The slicer config was built from scratch, and the SKR configs were from the very early days of Eddie's doc. I've got different pinouts but I've also added some new sections that would be beneficial for others to look at if they want.

### CODE REVIEW

File: firmware_configurations/klipper/IamPerrine/SKR_Fade_Config.txt [hunk 1]
```diff
@@ -0,0 +1,751 @@
...
+# Sensor Types
+#   "EPCOS 100K B57560G104F"
+#   "ATC Semitec 104GT-2"
+#   "NTC 100K beta 3950"
+#   "Honeywell 100K 135-104LAG-J01"
+#   "NTC 100K MGB18-104F39050L32" (Keenovo Heater Pad)
+#   "AD595"
+#   "PT100 INA826"
+
+#*# <---------------------- SAVE_CONFIG ---------------------->
```
Comment (FHeilmann) [inline firmware_configurations/klipper/IamPerrine/SKR_Fade_Config.txt]: This block will likely not conform to any other printer but your own. If someone chooses to use this config, they might miss that this block is present and get unexpected behavior. I suggest to remove them.

File: firmware_configurations/klipper/IamPerrine/SKR_Pancake_Config.txt [hunk 2]
```diff
@@ -0,0 +1,678 @@
...
+# Sensor Types
+#   "EPCOS 100K B57560G104F"
+#   "ATC Semitec 104GT-2"
+#   "NTC 100K beta 3950"
+#   "Honeywell 100K 135-104LAG-J01"
+#   "NTC 100K MGB18-104F39050L32" (Keenovo Heater Pad)
+#   "AD595"
+#   "PT100 INA826"
+
+#*# <---------------------- SAVE_CONFIG ---------------------->
```
Comment (FHeilmann) [inline firmware_configurations/klipper/IamPerrine/SKR_Pancake_Config.txt]: This block will likely not conform to any other printer but your own. If someone chooses to use this config, they might miss that this block is present and get unexpected behavior. I suggest to remove them.

File: slicer_configurations/PrusaSlicer/IamPerrine/Voron2_2.ini [hunk 3]
```diff
@@ -0,0 +1,235 @@
+# generated by PrusaSlicer 2.1.0+win64 on 2020-01-30 at 23:10:55 UTC
+avoid_crossing_perimeters = 0
+bed_custom_model = C:\\Users\\jakep\\OneDrive\\3D Printer\\VORON\\voron2_350bed.stl
```
Comment (FHeilmann) [inline slicer_configurations/PrusaSlicer/IamPerrine/Voron2_2.ini]: This hard link to the file on your machine won't work anywhere else, I suggest to remove it.

Comment (p3dstore) [reply slicer_configurations/PrusaSlicer/IamPerrine/Voron2_2.ini]: I'll make these changes. Do I need to push the update and make a new PR? Or will it update automatically? Sorry I'm a github noob still.

**Comment by FHeilmann** [comment]
@p3dstore any updates on this?

**Comment by p3dstore** [comment]
@FHeilmann Got your changes updated. Added Safe_Z_Home to the Fade Config too. Sorry for the delay, life got in the way.

**Pull request merged**